src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

```diff
@@ -4,39 +4,39 @@
         <body>
             <trans-unit id="1">
                 <source>This value should be false.</source>
-                <target>Vrednost treba da bude netačna.</target>
+                <target>Vrednost bi trebalo da bude netačna.</target>
```

Comment:
I think it was good before, I guess both are valid?

Reply:
The old one is understandable, but, if we want it by the rules, the new one is valid.